dashboard/common/utils.py

```diff
@@ -38,15 +40,15 @@ def get_git_version_info():
     return git_version
 
 
-def search_key_for_resource_value(my_dict, search_for):
+def search_key_for_resource_value(my_dict: Dict, search_for:str) -> Optional[str]:
     for key, value in my_dict.items():
         for resource_types in value["types"]:
             if search_for in resource_types:
                 return key
     return None
 
 
-def get_myla_globals(current_user):
+def get_myla_globals(current_user: HttpRequest.user) -> Dict[str, str] :
```

Comment:
Two things here:

1) When I log `current_user`, it says the type is `<class 'django.utils.functional.SimpleLazyObject'>`. Some [quick research](https://stackoverflow.com/questions/11314905/request-user-returns-a-simplelazyobject-how-do-i-wake-it) suggests that it doesn't become a `User` object until an attribute is accessed.
2) The type of `user_courses_info` is actually another dictionary, I think. So the return format should here should be `Dict[str, Any]` I guess?

Reply:
Yeah, looks like db_util.py isn't annotated. I can try to add some types there. 

I'm not sure about that SimpleLazyObject . . .

Reply:
I feel like HttpRequest.user is the correct type here and it's really what it looks like we're passing. There doesn't seem to be any issue so I think I'll just leave it for now. Some of these may have to be adjusted in the future.

Reply:
I'm just going to change this to pass and accept the request instead of the request.user.